packages/normalize/css/index.css

```diff
@@ -17,6 +17,55 @@
   font-style: italic;
 }
 
+@font-face {
+  font-family: 'DM Mono';
+  src: url('https://fonts.pluralsight.com/dm-mono/v1/dm-mono-light.ttf')
+    format('truetype');
+  font-display: swap;
+  font-weight: 300;
+  font-style: normal;
```

Comment:
Adding `font-feature-settings: 'ss05' on;` to the DM Mono `@font-face` declarations will switch to a better stylistic alternate of the character 'f'.